### DISCUSSION TITLE
Workflow deployment in non production slot via portal

### DISCUSSION THREAD

**Discussion by shashibhooshanzade-tomtom** [Azure/logicapps#1206] [Feature Ideas/Requests]
Hello Team,  Good day. Currently, we are trying our hands in the deployment slot preview version. We found that workflow deployment for non-production slot is not possible via Azure portal . Also the workflow is in read-only state in a non-production slot and testing is not possible. When will these features be generally available to make it similar to the Webapps deployment slot?

**Comment by cesarDevOp** [comment]
this is interesting, like Blue-green deployment

**Comment by smallj-univest** [comment]
This would be an amazing feature for lowering costs while having a mature dev pipeline for testing and acceptance.  I've just recently started vetting LogicApps as a replacement for some of our Mulesoft services and was extremely disappointed to find out that Deployment Slots can't be used as lower environments for testing.  Our only recourse to stick with LogicApps would be to create additional logic apps per env we want to support in our pipeline.

Please add this feature ASAP, it would really be a major defining feature against your competitors.
